pagefaultgames/pokerogue
# Enforce Consistent Spacing with ESLint's space-before-blocks and keyword-spacing Rules

## What are the changes?

Added the ESLint rules `space-before-blocks` and `keyword-spacing` to ensure consistent spacing before opening braces in blocks of code and around keywords. This enforces readability and maintains coding standards across the project.

## Why am I doing these changes?



This change is introduced to improve code readability and maintain consistency in the coding style. Consistent spacing before blocks and around keywords helps in understanding the structure of the code more clearly.

## What did change?


Before:
```javascript
if(condition){
  // code
}
```

After:
```javascript
if (condition) {
  // code
}
```
Added the following rules to the ESLint configuration:
```json
"space-before-blocks": ["error", "always"],
"keyword-spacing": ["error", { "before": true, "after": true }]
```

### Screenshots/Videos


Not applicable as this change does not directly affect the user interface.

## How to test the changes?




1. Checkout the branch with the ESLint configuration change.
2. Run the linter on the codebase.
3. Observe that ESLint now enforces a space before opening braces in blocks of code and around keywords.

## Checklist
  - [x] Have I provided screenshots/videos of the changes? (Not applicable)
